java/vector/src/test/java/org/apache/arrow/vector/complex/writer/TestComplexWriter.java

```diff
@@ -546,6 +548,54 @@ private void checkUnionList(ListVector listVector) {
     }
   }
 
+  @Test
+  public void testListMapType() {
+    try (ListVector listVector = ListVector.empty("list", allocator)) {
+      listVector.allocateNew();
+      UnionListWriter listWriter = new UnionListWriter(listVector);
+      MapWriter innerMapWriter = listWriter.map(true);
```

Comment:
can you verify the final vector has attribute `keysSorted` to `true`?

Reply:
Done